frontend/src/components/EmailPopUp.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <q-card class="bg-white text-white event-popup">
+    <q-bar class="email-bar">
+      <div class="text-h6">New Message</div>
+      <q-space />
+      <q-btn dense flat icon="close" v-close-popup></q-btn>
+    </q-bar>
+    <q-card-section>
+      <!-- Email form  -->
+      <q-form @submit="sendEmail" class="q-gutter-md">
+        <q-select
+          v-model="selectedEmails"
+          label="To"
+          :options="filterOptions"
+          multiple
+          use-chips
+          use-input
+          input-debounce="0"
+          @new-value="createRecipient"
+          @filter="filterEmails"
+          stack-label
+          lazy-rules
+          :rules="[
+            val =>
+              (val && val.length > 0) || 'Please enter at least one recipient'
+          ]"
+        />
+        <q-input
+          v-model="subject"
+          label="Subject"
+          lazy-rules
+          :rules="[
+            val =>
+              (val && val.length > 0) || 'Please enter a subject'
+          ]"
+        />
+        <q-input v-model="msg" type="textarea" />
+        <q-btn class="send-btn text-white" type="submit" label="Send" />
```

Comment:
One improvement for UX: disable the button when they click it so that they can't spam it and send more than one email. Add `:disabled="submitting"` to this button tag and add a variable called `submitting` to the `data()` section. Then at the start of the `sendEmail` function set `this.submitting = true` and set it back to false once everything is done